### PR TITLE
tzmigrate management command for prod

### PR DESCRIPTION
**Pull request by dannyroberts**
This is how I imagine using it in prod for my first real test https://gist.github.com/dannyroberts/29c77804030ff2e64cd1

### CODE REVIEW

File: corehq/apps/tzmigration/management/commands/tzmigrate.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
 from optparse import make_option
 from django.core.management.base import BaseCommand
+from casexml.apps.case.models import CommCareCase
```
Comment (dimagimon) [inline corehq/apps/tzmigration/management/commands/tzmigrate.py]: F401 'CommCareCase' imported but unused

File: corehq/apps/tzmigration/management/commands/tzmigrate.py [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 from corehq.apps.hqcase.dbaccessors import get_case_ids_in_domain
+from corehq.apps.tzmigration import set_migration_started, \
+    set_migration_complete
 from corehq.apps.tzmigration.timezonemigration import prepare_planning_db, \
     get_planning_db, get_planning_db_filepath, delete_planning_db, \
-    prepare_case_json, FormJsonDiff
+    prepare_case_json, FormJsonDiff, commit_plan
 from corehq.util.dates import iso_string_to_datetime
 from couchforms.dbaccessors import get_form_ids_by_type
+from couchforms.models import XFormInstance
```
Comment (dimagimon) [inline corehq/apps/tzmigration/management/commands/tzmigrate.py]: F401 'XFormInstance' imported but unused

**Comment by benrudolph** [comment]
I like the procedure, does ABORT completely rollback the commit or does it just stop where COMMIT left off

**Comment by dannyroberts** [comment]
The process is to run `--BEGIN` to stop accepting new forms, and then `--prepare` to run the migration but only save it to a local sqlite db, and then use the inspection tools to play around with the proposed migration (`--stats` to compare form/case counts between the the plan and the current data to make sure nothing went horribly wrong; `--show-diffs` to get a list of any diffs that aren't datetime format changes between current docs and the plan; `--play` to get dumped in an `ipdb` shell with some context variables already loaded), and then `--COMMIT` if you like what you see or `--ABORT` if you're concerned.

**Comment by dannyroberts** [comment]
So basically the way this is set up is that `--ABORT` just does nothing except tell HQ to start accepting forms again, and `--COMMIT` iterates through the planned changes and just saves them to couch.

**Comment by benrudolph** [comment]
well thought through, i'm signed off

**Pull request merged**